react/features/device-selection/components/DeviceSelection.js

```diff
@@ -372,7 +386,7 @@ class DeviceSelection extends AbstractDialogTab<Props, State> {
                 label: 'settings.selectCamera',
                 onSelect: selectedVideoInputId =>
                     super._onChange({ selectedVideoInputId }),
-                selectedDeviceId: this.props.selectedVideoInputId
+                selectedDeviceId: this.state.previewVideoInputId
```

Comment:
How did this ever work?

Reply:
When you are in a meeting it shows whatever is open currently in the meeting. But if you join audio or video muted, just shows "Select device".